lib/webmock/http_lib_adapters/http_rb/client.rb

```diff
@@ -4,7 +4,12 @@ class Client
 
     def perform(request, options)
       return __perform__(request, options) unless webmock_enabled?
-      WebMockPerform.new(request) { __perform__(request, options) }.exec
+
+      response = WebMockPerform.new(request) { __perform__(request, options) }.exec
+      response = options.features.inject(response) do |response, (_name, feature)|
+        feature.wrap_response(response)
+      end
+      response
```

Comment:
Nitpick (mostly to avoid shadowing):

```suggestion
      response = WebMockPerform.new(request) { __perform__(request, options) }.exec
      
      options.features.each { |_name, feature| response = feature.wrap_response(response) }
      
      response
```